browser/js/login/login.html

```diff
@@ -1,16 +1,31 @@
-<form id="login-form" name="loginForm" ng-submit="loginForm.$valid && sendLogin(login)">
+<div class="container">
```

Comment:
Works now! I moved the button out of the form tag while materialize-gridding which caused it to stop working.